content/agent/basic_agent_usage/amazonlinux.md

```diff
@@ -2,86 +2,161 @@
 title: Basic Agent Usage for Amazon Linux
 kind: documentation
 platform: Amazon Linux
-servicename: /etc/init.d/datadog-agent
-serviceinfoname: /etc/init.d/datadog-agent info
-configdirectory: /etc/dd-agent/
-logdirectory: /var/log/datadog/
-supervisorlog: /var/log/supervisor/datadog-supervisord.log
 aliases:
     - /guides/basic_agent_usage/amazonlinux/
 ---
 
 ## Overview
 
-This page outlines the basic functionality of the Datadog Agent.
+This page outlines the basic functionality of the Datadog Agent for Amazon linux.  
 If you haven't installed the Agent yet, instructions can be found
 [in the Datadog agent integration page](https://app.datadoghq.com/account/settings#agent/aws)
 
-The process to upgrade from the previous version of the agent is to re-run the installation.
+## Commands
 
-## Starting and Stopping the Agent
+Datadog Agent has some commands and only the _lifecycle commands_ (i.e. `start`/`stop`/`restart`/`status` on the Agent service) should be run with `sudo service`/`sudo initctl`/`sudo systemctl`, all other commands need to be run with the `datadog-agent` command.
 
-To manually start the Agent:
-```shell
-sudo /etc/init.d/datadog-agent start
-```
+{{% table responsive="true" %}}
+| Agent v5                                  |  Agent v6                          | Notes
+| ----------------------------------------------- | --------------------------------------- | ----------------------------- |
+| `sudo service datadog-agent start`              | `sudo service datadog-agent start`      | Start Agent as a service |
+| `sudo service datadog-agent stop`               | `sudo service datadog-agent stop`       | Stop Agent running as a service |
+| `sudo service datadog-agent restart`            | `sudo service datadog-agent restart`    | Restart Agent running as a service |
+| `sudo service datadog-agent status`             | `sudo service datadog-agent status`     | Status of Agent service |
+| `sudo service datadog-agent info`               | `sudo datadog-agent status`             | Status page of running Agent |
+| `sudo service datadog-agent flare`              | `sudo datadog-agent flare`              | Send flare |
+| `sudo service datadog-agent`                    | `sudo datadog-agent --help`             | Display command usage |
+| `sudo -u dd-agent -- dd-agent check <check_name>` | `sudo -u dd-agent --datadog-agent check <check_name>` | Run a check |
+{{% /table %}}
 
-To stop the Agent:
+More information about the metrics, events, and service checks for an [integrations](/integrations) can be retrieved with the check command:
 ```shell
-sudo /etc/init.d/datadog-agent stop
+sudo service datadog-agent check [integration]
```

Comment:
Two problems here:
1) this only works for A5
2) in the table above, the same command is listed as `sudo -u dd-agent -- dd-agent check <check_name>`

Reply:
👌